api/hooks/apiStatusHook/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function apiStatusHook() {
+  ////////////////
+  // API Status //
+  ////////////////
+  return {
+    getStatus: function () {
+      return new Promise(function (resolve, reject) {
+        Game.count().exec(function (error, game) {
+          if (error || game <= 0) {
+            return resolve(false);
+          }
+          return resolve(true);
+        });
+      });
```

Comment:
This is just the logic I had in mind, but can we move it into helpers, rather than a hook? There isn't much practical difference except both hooks and services are deprecated at this point. Here's the docs [on helpers](https://sailsjs.com/documentation/concepts/helpers). Same basic idea but you don't need a wrapping directory and the function guts get tucked into the `fn` attribute of a larger object that also has metadata like `friendlyName` and `description`. 

You can check `api/helpers/get-seasons-without-rankings` for a simple example.

Also would you mind rejiggering this to leverage `async/await`? The `exec()` method predates native promises in JS and `async/await` haha. So we haven't cleaned up the entire codebase in that regard (since I started on it before ES6 was a thing. So going forward, generally `async/await` are preferable imo (though we can still combine them with `Promise.all()` when doing multiple queries)

Reply:
I hope I did it right this time. I always get confused with promises and async/await 😂